webpack/assets/javascripts/react_app/components/HostDetails/EmptyState/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {
+  Title,
+  Button,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  EmptyStateSecondaryActions,
+} from '@patternfly/react-core';
+import { HddIcon } from '@patternfly/react-icons';
+import { translate as __ } from '../../../common/I18n';
+import { foremanUrl } from '../../../common/helpers';
+import { visit } from '../../../../foreman_navigation';
+
+const EmptyHostPage = () => (
+  <EmptyState variant="xl">
+    <EmptyStateIcon icon={HddIcon} />
+    <Title headingLevel="h4" size="lg">
+      {__('No host has found')}
```

Comment:
This feels a bit off. "No host was found" or "Host could not be found" would most likely be better. Although I'm not a native speaker so take it with a grain of salt.